README.md

```diff
@@ -46,7 +46,9 @@ If your app depends on the old unmigrated atmosphere packages than you also need
 
 ## The simplest possible test
 
-Basically, you run the tests with `gagarin` command within you project root.
+First run your meteor app with command `meteor`. 
```

Comment:
This is what I thought when I first run gagarin, but my tests shown something opposite. I saw in console that gagarin is building my app, but nothing happened later, it was frozen.